src/seedu/addressbook/data/person/Person.java

```diff
@@ -58,10 +64,22 @@ public Address getAddress() {
     public UniqueTagList getTags() {
         return new UniqueTagList(tags);
     }
+    
+    public int getSeqenceNumber() {
+    	return this.sequenceNumber;
+    }
 
     /**
      * Replaces this person's tags with the tags in the argument tag list.
      */
+    /**
+     * For tutorial 4 Activity 2 
+     * This method cannot be converted to a class-level method
+     * If the method become static, tags should also be static
+     * however, tags attribute will be unique for every single person (instance).
+     * If we set this method and tags as class-level,
+     * everyone in the AddressBook will have the same tag
+     */
```

Comment:
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.